article/app/controllers/LiveBlogController.scala

```diff
@@ -60,7 +61,16 @@ class LiveBlogController(
           case (minute: MinutePage, HtmlFormat) =>
             Future.successful(common.renderHtml(MinuteHtmlPage.html(minute), minute))
           case (blog: LiveBlogPage, HtmlFormat) => {
-            val remoteRendering = request.forceDCR && ActiveExperiments.isParticipating(LiveblogRendering)
+            // dcrCanRender is always false because right now blogs are not supported by DCR
+            // but we included this variable as an indication of what is going to be possible in the future
+            val dcrCanRender = false
```

Comment:
We have included this so that we can test the logic but at this point it is always false because at this point DCR serves zero liveblogs.

Reply:
+1 Like that we're laying the foundations for when we expand this :)